scripts/google_replicate.py

```diff
@@ -49,11 +53,15 @@ def bucket_exists(bucket_name):
     while tries < NUM_TRIES:
         try:
             return bucket.exists()
-        except Exception as e:
+        except BadRequest:
+            return False
+        except Forbidden as e:
+            logger.error("Bucket is not accessible {}. Detail {}".format(bucket_name, e))
+            return False
+        except Exception:
             time.sleep(300)
             tries += 1
-    
-    logger.error("Bucket is not accessible {}. Detail {}".format(bucket_name, e))
+
     return False
```

Comment:
add log here?

Reply:
Add more log in 887c6bab0bf09f49a15dfbd638d30cf15b54492e